ctk/idp/src/main/kotlin/io/kotlintest/provided/ProjectConfig.kt

```diff
@@ -14,16 +14,32 @@
 package io.kotlintest.provided
 
```

Comment:
❓ Should have asked this a while back...does this need to live in this package?

Reply:
Yup. https://github.com/kotlintest/kotlintest/blob/master/doc/reference.md#project-config